src/components/adminDashboard/components/eMetadata/controllers/educatorProfile.controller.js

```diff
@@ -0,0 +1,84 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.adminDashboard')
+        .controller('EducatorProfileController', function ($mdDialog, $timeout, userProfile, timezonesList, localTimezone, ZnkToastSrv, EMetadataService, $filter) {
+            'ngInject';
+            var self = this;
+            var translateFilter = $filter('translate');
+
+            self.timezonesList = timezonesList;
+            self.profileData = userProfile;
+            self.profileData.educatorTeachworksName = self.profileData.educatorTeachworksName || self.profileData.nickname;
```

Comment:
this scenario could not happen since the view passes the selected userProfile object, but to be sure, I added validation point.
` if(!userProfile){
                        $log.error('showEducatorProfile: userProfile is undefined');
                        return;
                    }`